tests/unit/src/services/submissionsService.spec.ts

```diff
@@ -1,128 +1,110 @@
+      expect(result).to.have.property("providerUserId", "validProviderUserId");
+      expect(result).to.have.property("providerOfficeId", "validProviderOfficeId");
+      expect(result).to.have.property("submissionTypeCode", "validSubmissionTypeCode");
+      expect(result).to.have.property("submissionDate", "validSubmissionDate");
+      expect(result).to.have.property("submissionPeriodStartDate", "validPeriodStart");
+      expect(result).to.have.property("submissionPeriodEndDate", "validPeriodEnd");
+      expect(result).to.have.property("scheduleId", "validScheduleId");
+
+      expect(result.claims).to.be.an("array");
+    });
 
     it("should throw an error when the transformer is used on null", () => {
-    
-    // Arrange - set up the mock data
-    const mockDataInput = null;
-
-    // Act - call transformer with the mock data
-    // Assert - check it is thrown
-
-    expect(() => transformSubmission(mockDataInput)).to.throw('Invalid submissions item: expected object'); 
-    })
-  })
+      // Arrange - set up the mock data
+      const mockDataInput = null;
+
+      // Act - call transformer with the mock data
+      // Assert - check it is thrown
+      expect(() => transformSubmission(mockDataInput)).to.throw(
+        "Invalid submissions item: expected object"
+      );
+    });
+  });
 
   describe("getSubmissions", () => {
-    let submissionsServiceStub: sinon.SinonStub;
-    let logSpy: sinon.SinonSpy;
-    let errorSpy: sinon.SinonSpy;
-
-    let middleware: AxiosInstanceWrapper;
-
-    let req: Partial<Request>;
+    let axiosStub: sinon.SinonStubbedInstance<AxiosInstance>;
+    let configureAxiosStub: sinon.SinonStub;
 
     beforeEach(() => {
-    // Reset the stub before each test
-
-    submissionsServiceStub = sinon.stub(submissionsService, "getSubmissions");
-    logSpy = sinon.spy(console, 'log')
-    errorSpy = sinon.spy(console, 'error')
+      // Reset the stub before each test
 
-    // Assign a mock or dummy value to middleware
-    middleware = {} as AxiosInstanceWrapper;
+      axiosStub = { get: sinon.stub() } as any;
+      configureAxiosStub = sinon
+        .stub(submissionsService as any, "configureAxiosInstance")
+        .returns(axiosStub);
     });
 
-    afterEach(() => { 
-    // Restore the stubs after each test
-    submissionsServiceStub.restore();
-    logSpy.restore();
-    errorSpy.restore();
+    afterEach(() => {
+      // Restore the stubs after each test
+      sinon.restore();
     });
 
-    it("should make a GET request correctly", async () => {
-    // Mocking resolved value of the API call
-    submissionsServiceStub.resolves(getSubmissionTestData)
-    
-    // faked API call
-    const response = await submissionsService.getSubmissions(middleware);
+    it("should call axios with the submissions endpoint", async () => {
+      // Arrange - Mocking resolved value of the API call
+      axiosStub.get.resolves({ data: [] });
```

Comment:
in this case define the empty data first with a name and it makes it clear. Comments are useful, especially to explain new functions or choices but otherwise they can make code harder to read.

`const mockEmptyData = `